### ISSUE TITLE
Lang-Tag not consistent

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/bigbluebutton#12576]
Descripton:
Using a Screenreader it is important to have the correct language specified. Otherwise for example the german UI is beeing read with english phornetic rules. 

Current behaviour
The dialogue to choose if I want to join with a microphone or just listen doesn't get the correct language assigned. There are other elements as well

expected behaviour:
Whatever the UI is set to should be respected so that users of screenreaders are able to understand the UI without changing the spoken language of their screenreaders manually. 

Thanks for helping.

**Comment by WestphalDenn** [comment]
closed due to the issue non-existent anymore.
